x-pack/plugins/cases/public/components/templates/form.test.tsx

```diff
@@ -257,9 +257,7 @@ describe('TemplateForm', () => {
       />
     );
 
-    const connectors = await screen.findByTestId('caseConnectors');
-
-    expect(await within(connectors).findByTestId('form-optional-field-label')).toBeInTheDocument();
```

Comment:
I removed the optional label from the template to be consistent with the create case form. If you want me to add it back let me know.